content/python/virtual_guest.py.md

```diff
@@ -0,0 +1,441 @@
+---
+title: "Working with Virtual Guest"
+description: "A few examples on interacting with Virtual Guest"
+date: "2020-07-09"
+classes: 
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Account"
+tags:
+    - "virtualservers"
+---
+
+### Create Virtual Server
```

Comment:
Change this example to create a VSI with a flavor, not CPU/Memory

Reply:
Done.